inspirehep/modules/forms/static/js/forms/experiments_typeahead.js

```diff
@@ -32,12 +32,14 @@ define([
       remote: {
         url: '/api/experiments?q=experimentautocomplete:%QUERY*',
         filter: function(response) {
-          return $.map(response.hits.hits, function(el) { return el });
-        }
-      },
+          return $.map(response.hits.hits, function(el) { return el }).sort(function(x, y) {
+          return x.metadata.experiment_names[0].title.localeCompare(y.metadata.experiment_names[0].title);
```

Comment:
Also, this line is doing three things, which can just as easily happen on three lines:
```javascript
var x_title = x.metadata.experiment_names[0].title,
    y_title = y.metadata.experiment_names[0].title;

return x_title.localeCompare(y_title);
```